src/NServiceBus.FileBasedRouting/FileBasedRoutingFeature.cs

```diff
@@ -112,7 +122,17 @@ public UpdateRoutingTask(Action updateRoutingCallback, TimeSpan routeFileUpdateI
 
             protected override Task OnStart(IMessageSession session)
             {
-                updateTimer = new Timer(state => updateRoutingCallback(), null, routeFileUpdateInterval, routeFileUpdateInterval);
+                updateTimer = new Timer(state =>
+                {
+                    try
+                    {
+                        updateRoutingCallback();
+                    }
+                    catch (Exception)
```

Comment:
why do we need the extra try/catch here? Can't we instead remove the throw in line 107 above? the method `updateRoutingCallback()` is wrapped in a try catch anyway.